kotlinx-coroutines-core/README.md

```diff
@@ -65,24 +86,36 @@ General-purpose coroutine builders, contexts, and helper functions.
 
 # Package kotlinx.coroutines.sync
 
-Synchronization primitives (mutex).
+Synchronization primitives (mutex and semaphore).
 
 # Package kotlinx.coroutines.channels
 
 Channels &mdash; non-blocking primitives for communicating a stream of elements between coroutines.
 
 # Package kotlinx.coroutines.flow
 
-Flow &mdash; asynchronous cold stream of elements.
+Flow &mdash; asynchronous cold and hot streams of elements.
 
 # Package kotlinx.coroutines.selects
 
-Select expression to perform multiple suspending operations simultaneously until one of them succeeds.
+Select &mdash; expressions that perform multiple suspending operations simultaneously until one of them succeeds.
 
 # Package kotlinx.coroutines.intrinsics
 
 Low-level primitives for finer-grained control of coroutines.
 
+# Package kotlinx.coroutines.future
+
+[JDK 8's `Future`](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletableFuture.html) support.
```

Comment:
```suggestion
[JDK 8's `CompletableFuture`](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletableFuture.html) support.
```

`Future` might be considered as https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/Future.html